scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -830,14 +830,14 @@ run_becktool() {
                 echo "Deleting core m0beck extra file $core_m0beck_file_count"
                 core_m0beck_last_file_name=$(cd /var/crash; ls -ltr | grep core-m0beck | awk '{print $9}' | tail -n 1)
                 echo "$core_m0beck_last_file_name"
-                rm /var/crash/$core_m0beck_last_file_name
+                rm /var/crash/"$core_m0beck_last_file_name"
```

Comment:
Or shall we:
```
rm "/var/crash/$core_m0beck_last_file_name"
```
?

Reply:
The purpose of using "$core_m0beck_last_file_name" was to prevent warning from codacy.
When "" is used for a variable, it prevents the expansion of any globbing or splitting characters that might be there in the variable.
So it won't be necessary to use it for the whole path.

Reply:
This path is changed to /var/log/crash in hw setups.
Update it accordingly here.

Reply:
@madhavemuri Added new commit a5e79db for the requested changes